homeassistant/components/idasen_desk/__init__.py

```diff
@@ -125,7 +116,7 @@ async def _async_stop(event: Event) -> None:
 async def _async_update_listener(hass: HomeAssistant, entry: ConfigEntry) -> None:
     """Handle options update."""
     data: DeskData = hass.data[DOMAIN][entry.entry_id]
-    if entry.title != data.device_info[ATTR_NAME]:
+    if entry.title != data.device_info.get(ATTR_NAME):
```

Comment:
Same comment from the previous PR

Reply:
You might want to store it as `data.name` so you can avoid the dict lookups and typing run around

Reply:
> You might want to store it as `data.name` so you can avoid the dict lookups and typing run around

I think I had that initially, but removed it to avoid duplicating the string.